### PR TITLE
Drop baseImageURL for ctlplane/vmsets

### PR DESCRIPTION
**Pull request by dprince**

### CODE REVIEW

File: api/v1beta1/openstackvmset_types.go [hunk 1]
```diff
@@ -31,12 +31,10 @@ type OpenStackVMSetSpec struct {
 	Memory uint32 `json:"memory"`
 	// root Disc size in GB
 	DiskSize uint32 `json:"diskSize"`
-	// Name of the VM base image used to setup the VMs
-	BaseImageURL string `json:"baseImageURL,omitempty"`
 	// StorageClass to be used for the disks
 	StorageClass string `json:"storageClass,omitempty"`
-	// BaseImageVolumeName Optional. If supplied will be used as the base volume for the VM instead of BaseImageURL.
-	BaseImageVolumeName string `json:"baseImageVolumeName,omitempty"`
+	// BaseImageVolumeName Optional. used as the base volume for the VM
```
Comment (abays) [inline api/v1beta1/openstackvmset_types.go:36]: Should probably take "optional" out of the description

Comment (stuggi) [reply api/v1beta1/openstackvmset_types.go:36]: yes, lets remove optional and ready to be merged

Comment (dprince) [reply api/v1beta1/openstackvmset_types.go]: got it

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**